.github/workflows/check-commit.yml

```diff
@@ -0,0 +1,31 @@
+name: Check Commit
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  check-signed-off-by:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+      - run: npm install
+
+      - run: |
+          ROLL_BACK_COUNT=$(( ${{ github.event.pull_request.commits }} - 1 ))
```

Comment:
Um.. I think this code is not working on GitHub Actions Push event. Because push event does not have `pull_request` property in github context. 

If you want to get commit message on Push event, you can use `github.event.head_commit.message` or `github.event.comits[0].message`.

<details>
<summary>pull_request context</summary>

```
GITHUB_CONTEXT: {
"event": {
    "action": "synchronize",
    "after": "0a7e30354ec1486e52940[541](https://github.com/jyoungyun/basic/runs/5531050845?check_suite_focus=true#step:3:541)94cc642a283f760a",
    "before": "cb94aa9d30620562b883883a5b0368a57fbcae3c",
    "number": 11,
    "pull_request": {
      "_links": {
        "comments": {
          "href": "https://api.github.com/repos/jyoungyun/basic/issues/11/comments"
        },
        "commits": {
          "href": "https://api.github.com/repos/jyoungyun/basic/pulls/11/commits"
        },
        "html": {
          "href": "https://github.com/jyoungyun/basic/pull/11"
        },
        "issue": {
          "href": "https://api.github.com/repos/jyoungyun/basic/issues/11"
        },
        "review_comment": {
          "href": "[https://api.github.com/repos/jyoungyun/basic/pulls/comments{/number}](https://api.github.com/repos/jyoungyun/basic/pulls/comments%7B/number%7D)"
        },
        "review_comments": {
          "href": "https://api.github.com/repos/jyoungyun/basic/pulls/11/comments"
        },
        "self": {
          "href": "https://api.github.com/repos/jyoungyun/basic/pulls/11"
        },
        "statuses": {
          "href": "https://api.github.com/repos/jyoungyun/basic/statuses/0a7e30354ec1486e5294054194cc642a283f760a"
        }
      },
      "active_lock_reason": null,
      "additions": 0,
      "assignee": null,
      "assignees": [],
      "author_association": "OWNER",
      "auto_merge": null,
      "base": {
        "label": "jyoungyun:master",
        "ref": "master",
        "repo": {
          "allow_auto_merge": false,
          "allow_forking": true,
          "allow_merge_commit": true,
          "allow_rebase_merge": true,
          "allow_squash_merge": true,
          "allow_update_branch": false,
          "archive_url": "[https://api.github.com/repos/jyoungyun/basic/{archive_format}{/ref}](https://api.github.com/repos/jyoungyun/basic/%7Barchive_format%7D%7B/ref%7D)",
          "archived": false,
          "assignees_url": "[https://api.github.com/repos/jyoungyun/basic/assignees{/user}](https://api.github.com/repos/jyoungyun/basic/assignees%7B/user%7D)",
          "blobs_url": "[https://api.github.com/repos/jyoungyun/basic/git/blobs{/sha}](https://api.github.com/repos/jyoungyun/basic/git/blobs%7B/sha%7D)",
          "branches_url": "[https://api.github.com/repos/jyoungyun/basic/branches{/branch}](https://api.github.com/repos/jyoungyun/basic/branches%7B/branch%7D)",
          "clone_url": "https://github.com/jyoungyun/basic.git",
          "collaborators_url": "[https://api.github.com/repos/jyoungyun/basic/collaborators{/collaborator}](https://api.github.com/repos/jyoungyun/basic/collaborators%7B/collaborator%7D)",
          "comments_url": "[https://api.github.com/repos/jyoungyun/basic/comments{/number}](https://api.github.com/repos/jyoungyun/basic/comments%7B/number%7D)",
          "commits_url": "[https://api.github.com/repos/jyoungyun/basic/commits{/sha}](https://api.github.com/repos/jyoungyun/basic/commits%7B/sha%7D)",
          "compare_url": "[https://api.github.com/repos/jyoungyun/basic/compare/{base}...{head}](https://api.github.com/repos/jyoungyun/basic/compare/%7Bbase%7D...%7Bhead%7D)",
          "contents_url": "[https://api.github.com/repos/jyoungyun/basic/contents/{+path}](https://api.github.com/repos/jyoungyun/basic/contents/%7B+path%7D)",
          "contributors_url": "https://api.github.com/repos/jyoungyun/basic/contributors",
          "created_at": "2020-09-22T02:53:02Z",
          "default_branch": "master",
          "delete_branch_on_merge": false,
          "deployments_url": "https://api.github.com/repos/jyoungyun/basic/deployments",
          "description": null,
          "disabled": false,
          "downloads_url": "https://api.github.com/repos/jyoungyun/basic/downloads",
          "events_url": "https://api.github.com/repos/jyoungyun/basic/events",
          "fork": false,
          "forks": 0,
          "forks_count": 0,
          "forks_url": "https://api.github.com/repos/jyoungyun/basic/forks",
          "full_name": "jyoungyun/basic",
          "git_commits_url": "https://api.github.com/repos/jyoungyun/basic/git/commits{/sha}",
          "git_refs_url": "https://api.github.com/repos/jyoungyun/basic/git/refs{/sha}",
          "git_tags_url": "https://api.github.com/repos/jyoungyun/basic/git/tags{/sha}",
          "git_url": "git://github.com/jyoungyun/basic.git",
          "has_downloads": true,
          "has_issues": true,
          "has_pages": false,
          "has_projects": true,
          "has_wiki": true,
          "homepage": null,
          "hooks_url": "https://api.github.com/repos/jyoungyun/basic/hooks",
          "html_url": "https://github.com/jyoungyun/basic",
          "id": 297518407,
          "is_template": false,
          "issue_comment_url": "https://api.github.com/repos/jyoungyun/basic/issues/comments{/number}",
          "issue_events_url": "https://api.github.com/repos/jyoungyun/basic/issues/events{/number}",
          "issues_url": "https://api.github.com/repos/jyoungyun/basic/issues{/number}",
          "keys_url": "https://api.github.com/repos/jyoungyun/basic/keys{/key_id}",
          "labels_url": "https://api.github.com/repos/jyoungyun/basic/labels{/name}",
          "language": "Vim script",
          "languages_url": "https://api.github.com/repos/jyoungyun/basic/languages",
          "license": null,
          "merges_url": "https://api.github.com/repos/jyoungyun/basic/merges",
          "milestones_url": "https://api.github.com/repos/jyoungyun/basic/milestones{/number}",
          "mirror_url": null,
          "name": "basic",
          "node_id": "MDEwOlJlcG9zaXRvcnkyOTc1MTg0MDc=",
          "notifications_url": "https://api.github.com/repos/jyoungyun/basic/notifications{?since,all,participating}",
          "open_issues": 5,
          "open_issues_count": 5,
          "owner": {
            "avatar_url": "https://avatars.githubusercontent.com/u/7223627?v=4",
            "events_url": "https://api.github.com/users/jyoungyun/events{/privacy}",
            "followers_url": "https://api.github.com/users/jyoungyun/followers",
            "following_url": "https://api.github.com/users/jyoungyun/following{/other_user}",
            "gists_url": "https://api.github.com/users/jyoungyun/gists{/gist_id}",
            "gravatar_id": "",
            "html_url": "https://github.com/jyoungyun",
            "id": 7223627,
            "login": "jyoungyun",
            "node_id": "MDQ6VXNlcjcyMjM2Mjc=",
            "organizations_url": "https://api.github.com/users/jyoungyun/orgs",
            "received_events_url": "https://api.github.com/users/jyoungyun/received_events",
            "repos_url": "https://api.github.com/users/jyoungyun/repos",
            "site_admin": false,
            "starred_url": "https://api.github.com/users/jyoungyun/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/jyoungyun/subscriptions",
            "type": "User",
            "url": "https://api.github.com/users/jyoungyun"
          },
          "private": false,
          "pulls_url": "https://api.github.com/repos/jyoungyun/basic/pulls{/number}",
          "pushed_at": "2022-03-14T01:19:54Z",
          "releases_url": "https://api.github.com/repos/jyoungyun/basic/releases{/id}",
          "size": 17,
          "ssh_url": "git@github.com:jyoungyun/basic.git",
          "stargazers_count": 0,
          "stargazers_url": "https://api.github.com/repos/jyoungyun/basic/stargazers",
          "statuses_url": "https://api.github.com/repos/jyoungyun/basic/statuses/{sha}",
          "subscribers_url": "https://api.github.com/repos/jyoungyun/basic/subscribers",
          "subscription_url": "https://api.github.com/repos/jyoungyun/basic/subscription",
          "svn_url": "https://github.com/jyoungyun/basic",
          "tags_url": "https://api.github.com/repos/jyoungyun/basic/tags",
          "teams_url": "https://api.github.com/repos/jyoungyun/basic/teams",
          "topics": [],
          "trees_url": "https://api.github.com/repos/jyoungyun/basic/git/trees{/sha}",
          "updated_at": "2021-09-30T08:30:11Z",
          "url": "https://api.github.com/repos/jyoungyun/basic",
          "visibility": "public",
          "watchers": 0,
          "watchers_count": 0
        },
        "sha": "66f36b76aa360c178ace71b7b76780d032c50513",
        "user": {
          "avatar_url": "https://avatars.githubusercontent.com/u/7223627?v=4",
          "events_url": "https://api.github.com/users/jyoungyun/events{/privacy}",
          "followers_url": "https://api.github.com/users/jyoungyun/followers",
          "following_url": "https://api.github.com/users/jyoungyun/following{/other_user}",
          "gists_url": "https://api.github.com/users/jyoungyun/gists{/gist_id}",
          "gravatar_id": "",
          "html_url": "https://github.com/jyoungyun",
          "id": 7223627,
          "login": "jyoungyun",
          "node_id": "MDQ6VXNlcjcyMjM2Mjc=",
          "organizations_url": "https://api.github.com/users/jyoungyun/orgs",
          "received_events_url": "https://api.github.com/users/jyoungyun/received_events",
          "repos_url": "https://api.github.com/users/jyoungyun/repos",
          "site_admin": false,
          "starred_url": "https://api.github.com/users/jyoungyun/starred{/owner}{/repo}",
          "subscriptions_url": "https://api.github.com/users/jyoungyun/subscriptions",
          "type": "User",
          "url": "https://api.github.com/users/jyoungyun"
        }
      },
      "body": "Add a\r\n\r\nSigned-off-by: Jiyoung Yun <jy910.yun@samsung.com>",
      "changed_files": 3,
      "closed_at": null,
      "comments": 0,
      "comments_url": "https://api.github.com/repos/jyoungyun/basic/issues/11/comments",
      "commits": 2,
      "commits_url": "https://api.github.com/repos/jyoungyun/basic/pulls/11/commits",
      "created_at": "2022-03-14T00:25:51Z",
      "deletions": 0,
      "diff_url": "https://github.com/jyoungyun/basic/pull/11.diff",
      "draft": false,
      "head": {
        "label": "jyoungyun:commit_test",
        "ref": "commit_test",
        "repo": {
          "allow_auto_merge": false,
          "allow_forking": true,
          "allow_merge_commit": true,
          "allow_rebase_merge": true,
          "allow_squash_merge": true,
          "allow_update_branch": false,
          "archive_url": "https://api.github.com/repos/jyoungyun/basic/{archive_format}{/ref}",
          "archived": false,
          "assignees_url": "https://api.github.com/repos/jyoungyun/basic/assignees{/user}",
          "blobs_url": "https://api.github.com/repos/jyoungyun/basic/git/blobs{/sha}",
          "branches_url": "https://api.github.com/repos/jyoungyun/basic/branches{/branch}",
          "clone_url": "https://github.com/jyoungyun/basic.git",
          "collaborators_url": "https://api.github.com/repos/jyoungyun/basic/collaborators{/collaborator}",
          "comments_url": "https://api.github.com/repos/jyoungyun/basic/comments{/number}",
          "commits_url": "https://api.github.com/repos/jyoungyun/basic/commits{/sha}",
          "compare_url": "https://api.github.com/repos/jyoungyun/basic/compare/{base}...{head}",
          "contents_url": "https://api.github.com/repos/jyoungyun/basic/contents/{+path}",
          "contributors_url": "https://api.github.com/repos/jyoungyun/basic/contributors",
          "created_at": "2020-09-22T02:53:02Z",
          "default_branch": "master",
          "delete_branch_on_merge": false,
          "deployments_url": "https://api.github.com/repos/jyoungyun/basic/deployments",
          "description": null,
          "disabled": false,
          "downloads_url": "https://api.github.com/repos/jyoungyun/basic/downloads",
          "events_url": "https://api.github.com/repos/jyoungyun/basic/events",
          "fork": false,
          "forks": 0,
          "forks_count": 0,
          "forks_url": "https://api.github.com/repos/jyoungyun/basic/forks",
          "full_name": "jyoungyun/basic",
          "git_commits_url": "https://api.github.com/repos/jyoungyun/basic/git/commits{/sha}",
          "git_refs_url": "https://api.github.com/repos/jyoungyun/basic/git/refs{/sha}",
          "git_tags_url": "https://api.github.com/repos/jyoungyun/basic/git/tags{/sha}",
          "git_url": "git://github.com/jyoungyun/basic.git",
          "has_downloads": true,
          "has_issues": true,
          "has_pages": false,
          "has_projects": true,
          "has_wiki": true,
          "homepage": null,
          "hooks_url": "https://api.github.com/repos/jyoungyun/basic/hooks",
          "html_url": "https://github.com/jyoungyun/basic",
          "id": 297518407,
          "is_template": false,
          "issue_comment_url": "https://api.github.com/repos/jyoungyun/basic/issues/comments{/number}",
          "issue_events_url": "https://api.github.com/repos/jyoungyun/basic/issues/events{/number}",
          "issues_url": "https://api.github.com/repos/jyoungyun/basic/issues{/number}",
          "keys_url": "https://api.github.com/repos/jyoungyun/basic/keys{/key_id}",
          "labels_url": "https://api.github.com/repos/jyoungyun/basic/labels{/name}",
          "language": "Vim script",
          "languages_url": "https://api.github.com/repos/jyoungyun/basic/languages",
          "license": null,
          "merges_url": "https://api.github.com/repos/jyoungyun/basic/merges",
          "milestones_url": "https://api.github.com/repos/jyoungyun/basic/milestones{/number}",
          "mirror_url": null,
          "name": "basic",
          "node_id": "MDEwOlJlcG9zaXRvcnkyOTc1MTg0MDc=",
          "notifications_url": "https://api.github.com/repos/jyoungyun/basic/notifications{?since,all,participating}",
          "open_issues": 5,
          "open_issues_count": 5,
          "owner": {
            "avatar_url": "https://avatars.githubusercontent.com/u/7223627?v=4",
            "events_url": "https://api.github.com/users/jyoungyun/events{/privacy}",
            "followers_url": "https://api.github.com/users/jyoungyun/followers",
            "following_url": "https://api.github.com/users/jyoungyun/following{/other_user}",
            "gists_url": "https://api.github.com/users/jyoungyun/gists{/gist_id}",
            "gravatar_id": "",
            "html_url": "https://github.com/jyoungyun",
            "id": 7223627,
            "login": "jyoungyun",
            "node_id": "MDQ6VXNlcjcyMjM2Mjc=",
            "organizations_url": "https://api.github.com/users/jyoungyun/orgs",
            "received_events_url": "https://api.github.com/users/jyoungyun/received_events",
            "repos_url": "https://api.github.com/users/jyoungyun/repos",
            "site_admin": false,
            "starred_url": "https://api.github.com/users/jyoungyun/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/jyoungyun/subscriptions",
            "type": "User",
            "url": "https://api.github.com/users/jyoungyun"
          },
          "private": false,
          "pulls_url": "https://api.github.com/repos/jyoungyun/basic/pulls{/number}",
          "pushed_at": "2022-03-14T01:19:54Z",
          "releases_url": "https://api.github.com/repos/jyoungyun/basic/releases{/id}",
          "size": 17,
          "ssh_url": "git@github.com:jyoungyun/basic.git",
          "stargazers_count": 0,
          "stargazers_url": "https://api.github.com/repos/jyoungyun/basic/stargazers",
          "statuses_url": "https://api.github.com/repos/jyoungyun/basic/statuses/{sha}",
          "subscribers_url": "https://api.github.com/repos/jyoungyun/basic/subscribers",
          "subscription_url": "https://api.github.com/repos/jyoungyun/basic/subscription",
          "svn_url": "https://github.com/jyoungyun/basic",
          "tags_url": "https://api.github.com/repos/jyoungyun/basic/tags",
          "teams_url": "https://api.github.com/repos/jyoungyun/basic/teams",
          "topics": [],
          "trees_url": "https://api.github.com/repos/jyoungyun/basic/git/trees{/sha}",
          "updated_at": "2021-09-30T08:30:11Z",
          "url": "https://api.github.com/repos/jyoungyun/basic",
          "visibility": "public",
          "watchers": 0,
          "watchers_count": 0
        },
        "sha": "0a7e30354ec1486e5294054194cc642a283f760a",
        "user": {
          "avatar_url": "https://avatars.githubusercontent.com/u/7223627?v=4",
          "events_url": "https://api.github.com/users/jyoungyun/events{/privacy}",
          "followers_url": "https://api.github.com/users/jyoungyun/followers",
          "following_url": "https://api.github.com/users/jyoungyun/following{/other_user}",
          "gists_url": "https://api.github.com/users/jyoungyun/gists{/gist_id}",
          "gravatar_id": "",
          "html_url": "https://github.com/jyoungyun",
          "id": 7223627,
          "login": "jyoungyun",
          "node_id": "MDQ6VXNlcjcyMjM2Mjc=",
          "organizations_url": "https://api.github.com/users/jyoungyun/orgs",
          "received_events_url": "https://api.github.com/users/jyoungyun/received_events",
          "repos_url": "https://api.github.com/users/jyoungyun/repos",
          "site_admin": false,
          "starred_url": "https://api.github.com/users/jyoungyun/starred{/owner}{/repo}",
          "subscriptions_url": "https://api.github.com/users/jyoungyun/subscriptions",
          "type": "User",
          "url": "https://api.github.com/users/jyoungyun"
        }
      },
      "html_url": "https://github.com/jyoungyun/basic/pull/11",
      "id": 878511097,
      "issue_url": "https://api.github.com/repos/jyoungyun/basic/issues/11",
      "labels": [],
      "locked": false,
      "maintainer_can_modify": false,
      "merge_commit_sha": "3f[610](https://github.com/jyoungyun/basic/runs/5531050845?check_suite_focus=true#step:3:610)57ec823f7c75f422aabb9a25ba473043c69",
      "mergeable": null,
      "mergeable_state": "unknown",
      "merged": false,
      "merged_at": null,
      "merged_by": null,
      "milestone": null,
      "node_id": "PR_kwDOEbvFR840XQP5",
      "number": 11,
      "patch_url": "https://github.com/jyoungyun/basic/pull/11.patch",
      "rebaseable": null,
      "requested_reviewers": [],
      "requested_teams": [],
      "review_comment_url": "https://api.github.com/repos/jyoungyun/basic/pulls/comments{/number}",
      "review_comments": 0,
      "review_comments_url": "https://api.github.com/repos/jyoungyun/basic/pulls/11/comments",
      "state": "open",
      "statuses_url": "https://api.github.com/repos/jyoungyun/basic/statuses/0a7e30354ec1486e5294054194cc642a283f760a",
      "title": "Add test",
      "updated_at": "2022-03-14T01:19:55Z",
      "url": "https://api.github.com/repos/jyoungyun/basic/pulls/11",
      "user": {
        "avatar_url": "https://avatars.githubusercontent.com/u/7223627?v=4",
        "events_url": "https://api.github.com/users/jyoungyun/events{/privacy}",
        "followers_url": "https://api.github.com/users/jyoungyun/followers",
        "following_url": "https://api.github.com/users/jyoungyun/following{/other_user}",
        "gists_url": "https://api.github.com/users/jyoungyun/gists{/gist_id}",
        "gravatar_id": "",
        "html_url": "https://github.com/jyoungyun",
        "id": 7223627,
        "login": "jyoungyun",
        "node_id": "MDQ6VXNlcjcyMjM2Mjc=",
        "organizations_url": "https://api.github.com/users/jyoungyun/orgs",
        "received_events_url": "https://api.github.com/users/jyoungyun/received_events",
        "repos_url": "https://api.github.com/users/jyoungyun/repos",
        "site_admin": false,
        "starred_url": "https://api.github.com/users/jyoungyun/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/jyoungyun/subscriptions",
        "type": "User",
        "url": "https://api.github.com/users/jyoungyun"
      }
    },
    "repository": {
      "allow_forking": true,
      "archive_url": "https://api.github.com/repos/jyoungyun/basic/{archive_format}{/ref}",
      "archived": false,
      "assignees_url": "https://api.github.com/repos/jyoungyun/basic/assignees{/user}",
      "blobs_url": "https://api.github.com/repos/jyoungyun/basic/git/blobs{/sha}",
      "branches_url": "https://api.github.com/repos/jyoungyun/basic/branches{/branch}",
      "clone_url": "https://github.com/jyoungyun/basic.git",
      "collaborators_url": "https://api.github.com/repos/jyoungyun/basic/collaborators{/collaborator}",
      "comments_url": "https://api.github.com/repos/jyoungyun/basic/comments{/number}",
      "commits_url": "https://api.github.com/repos/jyoungyun/basic/commits{/sha}",
      "compare_url": "https://api.github.com/repos/jyoungyun/basic/compare/{base}...{head}",
      "contents_url": "https://api.github.com/repos/jyoungyun/basic/contents/{+path}",
      "contributors_url": "https://api.github.com/repos/jyoungyun/basic/contributors",
      "created_at": "2020-09-22T02:53:02Z",
      "default_branch": "master",
      "deployments_url": "https://api.github.com/repos/jyoungyun/basic/deployments",
      "description": null,
      "disabled": false,
      "downloads_url": "https://api.github.com/repos/jyoungyun/basic/downloads",
      "events_url": "https://api.github.com/repos/jyoungyun/basic/events",
      "fork": false,
      "forks": 0,
      "forks_count": 0,
      "forks_url": "https://api.github.com/repos/jyoungyun/basic/forks",
      "full_name": "jyoungyun/basic",
      "git_commits_url": "[https://api.github.com/repos/jyoungyun/basic/git/commits{/sha}](https://api.github.com/repos/jyoungyun/basic/git/commits%7B/sha%7D)",
      "git_refs_url": "[https://api.github.com/repos/jyoungyun/basic/git/refs{/sha}](https://api.github.com/repos/jyoungyun/basic/git/refs%7B/sha%7D)",
      "git_tags_url": "[https://api.github.com/repos/jyoungyun/basic/git/tags{/sha}](https://api.github.com/repos/jyoungyun/basic/git/tags%7B/sha%7D)",
      "git_url": "git://github.com/jyoungyun/basic.git",
      "has_downloads": true,
      "has_issues": true,
      "has_pages": false,
      "has_projects": true,
      "has_wiki": true,
      "homepage": null,
      "hooks_url": "https://api.github.com/repos/jyoungyun/basic/hooks",
      "html_url": "https://github.com/jyoungyun/basic",
      "id": 297518407,
      "is_template": false,
      "issue_comment_url": "[https://api.github.com/repos/jyoungyun/basic/issues/comments{/number}](https://api.github.com/repos/jyoungyun/basic/issues/comments%7B/number%7D)",
      "issue_events_url": "[https://api.github.com/repos/jyoungyun/basic/issues/events{/number}](https://api.github.com/repos/jyoungyun/basic/issues/events%7B/number%7D)",
      "issues_url": "[https://api.github.com/repos/jyoungyun/basic/issues{/number}](https://api.github.com/repos/jyoungyun/basic/issues%7B/number%7D)",
      "keys_url": "https://api.github.com/repos/jyoungyun/basic/keys{/key_id}",
      "labels_url": "https://api.github.com/repos/jyoungyun/basic/labels{/name}",
      "language": "Vim script",
      "languages_url": "https://api.github.com/repos/jyoungyun/basic/languages",
      "license": null,
      "merges_url": "https://api.github.com/repos/jyoungyun/basic/merges",
      "milestones_url": "https://api.github.com/repos/jyoungyun/basic/milestones{/number}",
      "mirror_url": null,
      "name": "basic",
      "node_id": "MDEwOlJlcG9zaXRvcnkyOTc1MTg0MDc=",
      "notifications_url": "https://api.github.com/repos/jyoungyun/basic/notifications{?since,all,participating}",
      "open_issues": 5,
      "open_issues_count": 5,
      "owner": {
        "avatar_url": "https://avatars.githubusercontent.com/u/7223[627](https://github.com/jyoungyun/basic/runs/5531050845?check_suite_focus=true#step:3:627)?v=4",
        "events_url": "https://api.github.com/users/jyoungyun/events{/privacy}",
        "followers_url": "https://api.github.com/users/jyoungyun/followers",
        "following_url": "https://api.github.com/users/jyoungyun/following{/other_user}",
        "gists_url": "https://api.github.com/users/jyoungyun/gists{/gist_id}",
        "gravatar_id": "",
        "html_url": "https://github.com/jyoungyun",
        "id": 7223627,
        "login": "jyoungyun",
        "node_id": "MDQ6VXNlcjcyMjM2Mjc=",
        "organizations_url": "https://api.github.com/users/jyoungyun/orgs",
        "received_events_url": "https://api.github.com/users/jyoungyun/received_events",
        "repos_url": "https://api.github.com/users/jyoungyun/repos",
        "site_admin": false,
        "starred_url": "https://api.github.com/users/jyoungyun/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/jyoungyun/subscriptions",
        "type": "User",
        "url": "https://api.github.com/users/jyoungyun"
      },
      "private": false,
      "pulls_url": "https://api.github.com/repos/jyoungyun/basic/pulls{/number}",
      "pushed_at": "2022-03-14T01:19:54Z",
      "releases_url": "https://api.github.com/repos/jyoungyun/basic/releases{/id}",
      "size": 17,
      "ssh_url": "git@github.com:jyoungyun/basic.git",
      "stargazers_count": 0,
      "stargazers_url": "https://api.github.com/repos/jyoungyun/basic/stargazers",
      "statuses_url": "https://api.github.com/repos/jyoungyun/basic/statuses/{sha}",
      "subscribers_url": "https://api.github.com/repos/jyoungyun/basic/subscribers",
      "subscription_url": "https://api.github.com/repos/jyoungyun/basic/subscription",
      "svn_url": "https://github.com/jyoungyun/basic",
      "tags_url": "https://api.github.com/repos/jyoungyun/basic/tags",
      "teams_url": "https://api.github.com/repos/jyoungyun/basic/teams",
      "topics": [],
      "trees_url": "https://api.github.com/repos/jyoungyun/basic/git/trees{/sha}",
      "updated_at": "2021-09-30T08:30:11Z",
      "url": "https://api.github.com/repos/jyoungyun/basic",
      "visibility": "public",
      "watchers": 0,
      "watchers_count": 0
    },
    "sender": {
      "avatar_url": "https://avatars.githubusercontent.com/u/7223627?v=4",
      "events_url": "[https://api.github.com/users/jyoungyun/events{/privacy}](https://api.github.com/users/jyoungyun/events%7B/privacy%7D)",
      "followers_url": "https://api.github.com/users/jyoungyun/followers",
      "following_url": "[https://api.github.com/users/jyoungyun/following{/other_user}](https://api.github.com/users/jyoungyun/following%7B/other_user%7D)",
      "gists_url": "[https://api.github.com/users/jyoungyun/gists{/gist_id}](https://api.github.com/users/jyoungyun/gists%7B/gist_id%7D)",
      "gravatar_id": "",
      "html_url": "https://github.com/jyoungyun",
      "id": 7223627,
      "login": "jyoungyun",
      "node_id": "MDQ6VXNlcjcyMjM2Mjc=",
      "organizations_url": "https://api.github.com/users/jyoungyun/orgs",
      "received_events_url": "https://api.github.com/users/jyoungyun/received_events",
      "repos_url": "https://api.github.com/users/jyoungyun/repos",
      "site_admin": false,
      "starred_url": "[https://api.github.com/users/jyoungyun/starred{/owner}{/repo}](https://api.github.com/users/jyoungyun/starred%7B/owner%7D%7B/repo%7D)",
      "subscriptions_url": "https://api.github.com/users/jyoungyun/subscriptions",
      "type": "User",
      "url": "https://api.github.com/users/jyoungyun"
    }
  },
```

</details>

Reply:
Q. So, is it resolved?

Reply:
Yes, by removing `push` event :D